src/lib/__tests__/all.test.js

```diff
@@ -1,34 +1,26 @@
-import all from "lib/all"
+import { allViaLoader } from "lib/all"
 
 describe("all", () => {
-  afterEach(() => {
-    all.__ResetDependency__("total")
-    all.__ResetDependency__("gravity")
-  })
-
-  it("fans out all the request", () => {
-    const gravity = sinon.stub()
-
-    all.__Rewire__("total", sinon.stub().returns(Promise.resolve(120)))
-    all.__Rewire__("gravity", gravity.returns(Promise.resolve([{}])))
+  it("fans out all the requests", () => {
+    const loader = jest
+      .fn()
+      .mockReturnValueOnce(
```

Comment:
The first call is to retrieve the count, and then we fan out all required pages (for simplicity just resolving with an empty body as this helper doesn't care _what_ is resolved).